src/lang/english.txt

```diff
@@ -1556,6 +1556,8 @@ STR_CONFIG_SETTING_SEMAPHORE_BUILD_BEFORE_DATE                  :Automatically b
 STR_CONFIG_SETTING_SEMAPHORE_BUILD_BEFORE_DATE_HELPTEXT         :Set the year when electric signals will be used for tracks. Before this year, non-electric signals will be used (which have the exact same function, but different looks)
 STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI                            :Enable the signal GUI: {STRING2}
 STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_HELPTEXT                   :Display a window for choosing signal types to build, instead of only window-less signal-type rotation with Ctrl+clicking on built signals
+STR_CONFIG_SETTING_SHOW_BLOCKSIGNALS                            :Enable building block signals: {STRING2}
+STR_CONFIG_SETTING_SHOW_BLOCKSIGNALS_HELPTEXT                   :Show the tools for building block signals and pre-signals. Usage of block signals and path signals is discouraged, since path signals usually solve the same problems better. Existing block signals will continue working.
```

Comment:
Not a fan of this, reads as very disparaging to the user. Something like "Block & pre(not path!) signals are intended for advanced users only"